frontend/src/components/pages/CampOverview/index.tsx

```diff
@@ -38,7 +38,7 @@ const CampOverviewPage = (): JSX.Element => {
     volunteers: "",
     campPhotoUrl: "",
   });
-  const numSessions = camp.campSessions.length;
+  const numSessions = camp.campSessions?.length;
```

Comment:
Is this a fix related to this bug ticket or just a fix in general?

Reply:
fix in general!